src/content/es/2020/http2.md

```diff
@@ -0,0 +1,690 @@
+        <td class="numeric">1</td>
+        <td class="numeric">36.34</td>
+        <td class="numeric">1</td>
+        <td class="numeric">0.06</td>
+      </tr>
+      <tr>
+        <td class="numeric">50</td>
+        <td class="numeric">3</td>
+        <td class="numeric">183.83</td>
+        <td class="numeric">2</td>
+        <td class="numeric">24.06</td>
+      </tr>
+      <tr>
+        <td class="numeric">75</td>
+        <td class="numeric">10</td>
+        <td class="numeric">225.41</td>
+        <td class="numeric">5</td>
+        <td class="numeric">204.65</td>
+      </tr>
+      <tr>
+        <td class="numeric">90</td>
+        <td class="numeric">12</td>
+        <td class="numeric">351.05</td>
+        <td class="numeric">18</td>
+        <td class="numeric">453.57</td>
+      </tr>
+    </tbody>
+  </table>
+  <figcaption>{{ figure_link(caption="Distribución de activos empujados en dispositivos de escritorio.", sheets_gid="698874709", sql_file="number_of_h2_and_h3_pushed_resources_and_bytes_transferred.sql") }}</figcaption>
+</figure>
+
+<figure>
+  <table>
+    <thead>
+      <tr>
+        <th>Percentil</th>
+        <th>HTTP/2</th>
+        <th>Tamaño (KB)</th>
+        <th>gQUIC</th>
+        <th>Tamaño (KB)</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td class="numeric">10</td>
+        <td class="numeric">1</td>
+        <td class="numeric">15.48</td>
+        <td class="numeric">1</td>
+        <td class="numeric">0.06</td>
+      </tr>
+      <tr>
+        <td class="numeric">25</td>
+        <td class="numeric">1</td>
+        <td class="numeric">36.34</td>
+        <td class="numeric">1</td>
+        <td class="numeric">0.06</td>
+      </tr>
+      <tr>
+        <td class="numeric">50</td>
+        <td class="numeric">3</td>
+        <td class="numeric">183.83</td>
+        <td class="numeric">2</td>
+        <td class="numeric">24.06</td>
+      </tr>
+      <tr>
+        <td class="numeric">75</td>
+        <td class="numeric">10</td>
+        <td class="numeric">225.41</td>
+        <td class="numeric">5</td>
+        <td class="numeric">204.65</td>
+      </tr>
+      <tr>
+        <td class="numeric">90</td>
+        <td class="numeric">12</td>
+        <td class="numeric">351.05</td>
+        <td class="numeric">18</td>
+        <td class="numeric">453.57</td>
+      </tr>
+    </tbody>
+  </table>
+  <figcaption>{{ figure_link(caption="Distribución de activos empujados en dispositivos móviles.", sheets_gid="698874709", sql_file="number_of_h2_and_h3_pushed_resources_and_bytes_transferred.sql") }}</figcaption>
+</figure>
+
+Al observar la frecuencia de empuje por tipo de contenido en la Figura 22.20, vemos que el 90% de las páginas empujan scripts y el 56% empujan CSS. Esto tiene sentido, ya que pueden ser archivos pequeños que normalmente se encuentran en la ruta crítica para representar una página.
+
+{# TODO(analysts): Should this chart differentiate desktop and mobile? #}
```

Comment:
Looks like this was done. Can you remove this line from Spanish and English version?

```suggestion
```